todo_app/data/test_view_model.py

```diff
@@ -0,0 +1,59 @@
+from todo_app.data.item import Item
+from todo_app.data.view_model import ViewModel 
+import pytest
+
+# Arrange
+@pytest.fixture
+def do_items_list(): 
+    return [
+        Item(1, "My Todo Item 1", "To Do")
+    ]
+ 
+@pytest.fixture   
+def doing_items_list():
+    return [Item(1, "My Doing Item 1", "Doing"),Item(2, "My Doing Item 2", "Doing"),Item(3, "My Doing Item 3", "Doing")]
+
+@pytest.fixture    
+def done_items_list():
+    return [
+        Item(1, "My Done Item 1", "Done")
+    ]    
+
+def test_do_items_property_only_shows_do_items(do_items_list):
+    
+    # Arrange
+    view_model = ViewModel(do_items_list, doing_items_list, done_items_list)
```

Comment:
Hi Bogdan, Python wouldn't let me concatenate functions but instead of finding a way around this, I used the same logic from comments made in a previous module and simplified the set up by just having a one fixture but with mixture of "list types" in that fixture. Tests are all now passing again. Thanks again for all your help here.